features/support/fs.rs

```diff
@@ -165,13 +167,23 @@ fn create_features(dir: TempDir) -> io::Result<TempDir> {
 }
 
 fn bootstrap_target(dir: TempDir) -> io::Result<TempDir> {
-  // TODO: consider a solution using standard Rust
-  Command::new("cp")
-    .arg(env::current_dir().unwrap().join("target"))
-    .arg(dir.path().join("target"))
-    .arg("-r")
-    .output()
-    .map(|_| dir)
+  copy_recursively(&env::current_dir().unwrap().join("target"), &dir.path().join("target")).map(|_| dir)
+}
+
+fn copy_recursively(origin_base: &Path, target_base: &Path) -> io::Result<u64> {
+  WalkDir::new(origin_base).into_iter().filter_map(|e| e.ok()).fold(Ok(0), |sum, entry| {
+    if entry.path().is_file() {
+      let relative = entry.path().strip_prefix(origin_base).unwrap();
+      let target = target_base.join(relative);
+      // println!("orig: {}, target: {}", entry.path().to_str().unwrap(), target.to_str().unwrap());
+      // println!("orig: {}, target: {}", entry.path().is_file(), target.is_file());
+      // println!("Copying file!");
+      copy(&entry.path(), &target).map(|_| sum.unwrap() + 1)
```

Comment:
Just took a look at this. Looks like it works, but it needs the parent directory to be created before the file can be copied.

above this line add:

```rust
let _ = create_dir_all(target.parent().unwrap());
```

I'm ignoring the result because we're going to try to create the dir many times, and it'll fail if its already created. If you've got a better way to do this, that would be appreciated.

Making this change made the tests pass on my end.